sqlmesh/core/loader.py

```diff
@@ -261,6 +263,7 @@ def _load_scripts(self) -> t.Tuple[MacroRegistry, JinjaMacroRegistry]:
                         else macro_file_mtime
                     )
 
+            macro_definitions = []
```

Comment:
Let's be more specific with the name. There are too many different kinds of macros in this method